includes/modules/pages/account/header_php.php

```diff
@@ -31,44 +31,9 @@
 require(DIR_WS_MODULES . zen_get_module_directory('require_languages.php'));
 
 $breadcrumb->add(NAVBAR_TITLE);
-$orders_query = "SELECT o.orders_id, o.date_purchased, o.delivery_name,
-                        o.delivery_country, o.billing_name, o.billing_country,
-                        ot.text as order_total, s.orders_status_name
-                 FROM   " . TABLE_ORDERS . " o, " . TABLE_ORDERS_TOTAL . "  ot, " . TABLE_ORDERS_STATUS . " s
-                 WHERE  o.customers_id = :customersID
-                 AND    o.orders_id = ot.orders_id
-                 AND    ot.class = 'ot_total'
-                 AND    s.orders_status_id = 
-                          (SELECT orders_status_id FROM " . TABLE_ORDERS_STATUS_HISTORY . " osh 
-                           WHERE osh.orders_id = o.orders_id AND osh.customer_notified >= 0 ORDER BY osh.date_added DESC LIMIT 1)
-                 AND   s.language_id = :languagesID
-                 ORDER BY orders_id DESC LIMIT 3";
 
-$orders_query = $db->bindVars($orders_query, ':customersID', $_SESSION['customer_id'], 'integer');
-$orders_query = $db->bindVars($orders_query, ':languagesID', $_SESSION['languages_id'], 'integer');
-$orders = $db->Execute($orders_query);
-
-$ordersArray = array();
-while (!$orders->EOF) {
-  if (zen_not_null($orders->fields['delivery_name'])) {
-    $order_name = $orders->fields['delivery_name'];
-    $order_country = $orders->fields['delivery_country'];
-  } else {
-    $order_name = $orders->fields['billing_name'];
-    $order_country = $orders->fields['billing_country'];
-  }
-
-  $ordersArray[] = array('orders_id'=>$orders->fields['orders_id'],
-  'date_purchased'=>$orders->fields['date_purchased'],
-  'order_name'=>$order_name,
-  'order_country'=>$order_country,
-  'orders_status_name'=>$orders->fields['orders_status_name'],
-  'order_total'=>$orders->fields['order_total']
-  );
-
-  $orders->MoveNext();
-}
+$customer = new Customer;
```

Comment:
Customer() ?

Reply:
Ya, I'm leaving it off when not passing any args.